test/quantization/fx/test_equalize_fx.py

```diff
@@ -135,220 +212,360 @@ def test_input_weight_eq_observer(self, input_qdtype, input_qscheme, weight_qdty
+        ]
+
+        linearAdd_node_list = [
+            ns.call_function(torch.mul),
+            ns.call_function(torch.quantize_per_tensor),
+            ns.call_module(nnq.Linear),
+            ns.call_method('dequantize'),
+            ns.call_function(torch.add),
+            ns.call_function(torch.mul),
+            ns.call_function(torch.quantize_per_tensor),
+            ns.call_module(nnq.Linear),
+            ns.call_method('dequantize')
+        ]
+
+        linear2_node_list = [
+            ns.call_function(torch.mul),
+            ns.call_function(torch.quantize_per_tensor),
+            ns.call_module(nnq.Linear),
+            ns.call_module(nnq.Linear),
+            ns.call_method('dequantize')
+        ]
+
+        functionalLinear_node_list = [
+            ns.call_function(torch.mul),
+            ns.call_function(torch.quantize_per_tensor),
+            ns.call_function(torch.ops.quantized.linear),
+            ns.call_method('dequantize')
+        ]
+
+        functionalLinearAdd_node_list = [
+            ns.call_function(torch.mul),
+            ns.call_function(torch.quantize_per_tensor),
+            ns.call_function(torch.ops.quantized.linear),
+            ns.call_method('dequantize'),
+            ns.call_function(torch.add),
+            ns.call_function(torch.mul),
+            ns.call_function(torch.quantize_per_tensor),
+            ns.call_function(torch.ops.quantized.linear),
+            ns.call_method('dequantize')
+        ]
+
+        functionalLinear2_node_list = [
+            ns.call_function(torch.mul),
+            ns.call_function(torch.quantize_per_tensor),
+            ns.call_function(torch.ops.quantized.linear),
+            ns.call_function(torch.ops.quantized.linear),
+            ns.call_method('dequantize')
+        ]
+
+        tests = [(SingleLayerLinearModel, linear_node_list),
+                 (LinearAddModule, linearAdd_node_list),
+                 (TwoLayerLinearModel, linear2_node_list),
+                 (FunctionalLinearModule, functionalLinear_node_list),
+                 (FunctionalLinearAddModule, functionalLinearAdd_node_list),
+                 (FunctionalLinear2Module, functionalLinear2_node_list)]
+
+        torch.manual_seed(0)
+        x = torch.rand((5, 5))
+        for (M, node_list) in tests:
+            # No equalization
+            m = M().eval()
+            prepared = prepare_fx(copy.deepcopy(m), qconfig_dict)
+            prepared_model = copy.deepcopy(prepared)
+            prepared(x)
+            quantized_model = convert_fx(prepared)
+
+            # Equalization
+            prepared = prepare_fx(copy.deepcopy(m), qconfig_dict, equalization_qconfig_dict=default_equalization_qconfig_dict)
+            prepared(x)
+            equalized_model = _convert_equalization_ref(prepared)
+
+            # Check that the graph before quantization is pretty much the same
+            self.assertTrue(self.check_orig_and_eq_graphs(prepared_model, equalized_model))
```

Comment:
Do we still need this function to check if we are using `checkGraphModuleNodes`?

Reply:
I guess not, I can remove that then!